package.json

```diff
@@ -24,21 +24,22 @@
   ],
   "devDependencies": {
     "@babel/core": "7.8.7",
-    "@babel/runtime": "7.11.2",
     "@babel/plugin-proposal-class-properties": "7.8.3",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
     "@babel/preset-env": "7.8.7",
     "@babel/preset-react": "7.8.3",
+    "@babel/runtime": "7.11.2",
     "@geosolutions/acorn-jsx": "4.0.2",
     "@geosolutions/jsdoc": "3.4.4",
     "@geosolutions/mocha": "6.2.1-3",
-    "@mapstore/eslint-config-mapstore": "1.0.1",
+    "@mapstore/eslint-config-mapstore": "1.0.2",
     "axios-mock-adapter": "1.16.0",
     "babel-istanbul-loader": "0.1.0",
     "babel-loader": "8.0.5",
     "babel-plugin-add-module-exports": "0.1.4",
     "babel-plugin-object-assign": "1.2.1",
     "babel-plugin-react-transform": "2.0.2",
+    "babel-plugin-transform-imports": "^2.0.0",
```

Comment:
```suggestion
    "babel-plugin-transform-imports": "2.0.0",
```

Reply:
Already committed the same suggestion from @MV88 .